operator/pkg/migrations/registry.go

```diff
@@ -110,6 +110,9 @@ func RegistryData(ctx context.Context) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      registry.RegistryDataMigrationCompleteSecretName,
 			Namespace: registry.RegistryNamespace(),
+			Labels: map[string]string{
+				"replicated.com/disaster-recovery": "ec-install",
```

Comment:
why? the `ec-install` label was specific to the `installation` CR/CRD. can we use a different label for this?

Reply:
also, why is it needed now but not before?

Reply:
this fixes a bug that doesnt seem to have any impact but the registry gets removed entirely from the clusterconfig after an upgrade of an ha cluster. this is because the registry migration status condition gets set to false in the new installation so the ha registry is never added back. This would have impact if we were ever to try to change a registry value as k0s is no longer controlling this object.

see related changes:

https://github.com/replicatedhq/embedded-cluster/pull/1238/files#diff-ec385ffb8fa0ab6903f92cdf907ca63ed0df571f73ca6c1edaabc7395fc1771eR95-R101

https://github.com/replicatedhq/embedded-cluster/pull/1238/files#diff-1595f9a51283933adb1a95f0d4a42f0576e783a1932c0d0da27fa1e5ca5628b3R35

Reply:
i see. any reason not to have it use the registry DR label instead of the ec-install one?

Reply:
i believe if i use the registry component it is too late and the operator will revert the registry back to non-ha in the clusterconfig. i would be wrong